aten/src/ATen/native/native_functions.yaml

```diff
@@ -37,10 +37,11 @@
   use_c10_dispatcher: full
   variants: function
 
-- func: backward(Tensor self, Tensor? gradient=None, bool keep_graph=False, bool create_graph=False) -> void
+- func: backward(Tensor self, Tensor? gradient=None, bool keep_graph=False, bool create_graph=False) -> ()
   variants: method
 
-- func: set_data(Tensor(a!) self, Tensor new_data) -> void
+- func: set_data(Tensor(a!) self, Tensor new_data) -> Tensor(a!)
```

Comment:
@yf225 @pbelevich This makes `set_data` support method chaining. Seem good to you? (I also kind of wonder why we called this `set_data` and not `data_`.)

Reply:
Yes it sounds great. We were following the naming for the `Variable::set_data` method, but maybe we should change it to `data_` instead.